_posts/2018-11-08-Rust-1.30.1.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Announcing Rust 1.30.1"
+author: The Rust Release Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.30.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.30.1 is as easy as:
+
+```
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.30.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1301-2018-11-08
+
+## What's in 1.30.1 stable
+
+This patch release fixes broken Cargo progress bars in MSYS terminals on
+Windows by capping the progress bar width to 60 columns. This doesn't affect
+other terminal emulators (like `cmd.exe` or PowerShell).
+
+This patch release also fixes a compiler panic that happened while building the
+docs of some crates in Rust 1.30.0.
```

Comment:
I think it'd be nice to add one more sentence about why this was severe enough to consider a point release.

also, both of these should have links to the PRs